test/modules/standard/math/promotion/gen/NOTEST

```diff
@@ -0,0 +1 @@
+This directory contains the test generation scripts. (no tests)
```

Comment:
You might want to choose a different name for this directory, since gen is in the gitignore list.